geometry_msgs/Pose pose
bool gripper
---
moveit_msgs/MoveItErrorCodes ErrorCode
